lib/ruby_lsp/utils.rb

```diff
@@ -4,6 +4,7 @@
 module RubyLsp
   # Used to indicate that a request shouldn't return a response
   VOID = T.let(Object.new.freeze, Object)
+  WORKSPACE_URI = T.let("file://#{Dir.pwd}", String)
```

Comment:
Should we `.freeze` this string?

Reply:
Oh, this was actually fixed already. The reason it fails for us is because we aim RuboCop for 2.7 (the earliest version we support).

Not actually a bug, just our configs.